src/devices/Common/tests/GeographicPositionTest.cs

```diff
@@ -39,10 +40,31 @@ public void EqualityGeographicPosition()
         [Fact]
         public void DefaultFormatting()
         {
-            Assert.Equal("10° 00' 00.00\"N / 32° 30' 00.00\"E Ellipsoidal Height 12345m", new GeographicPosition(10.0, 32.5, 12345).ToString());
-            Assert.Equal("111° 06' 08.42\"S / 32° 59' 15.36\"W Ellipsoidal Height 200m", new GeographicPosition(-111.10234, -32.9876, 200).ToString());
-            Assert.Equal("11° 00' 00.00\"S / 33° 00' 00.00\"E Ellipsoidal Height 200m", new GeographicPosition(-10.99999999999999999, 32.999999999999999, 200).ToString());
+            Assert.Equal("10° 00' 00.00\"N 032° 30' 00.00\"E Ellipsoidal Height 12345m", new GeographicPosition(10.0, 32.5, 12345).ToString());
+            Assert.Equal("11° 06' 08.42\"S 132° 59' 15.36\"W Ellipsoidal Height 200m", new GeographicPosition(-11.10234, -132.9876, 200).ToString());
+            Assert.Equal("11° 00' 00.00\"S 033° 00' 00.00\"E Ellipsoidal Height 200m", new GeographicPosition(-10.99999999999999999, 32.999999999999999, 200).ToString());
         }
 
+        [Theory]
+        [InlineData("10.000° 23.500°", 10.0, 23.5, 12345, "D3 D3")]
+        [InlineData("10.000°N 23.500°E", 10.0, 23.5, 12345, "D3N D3E")]
+        [InlineData("-20.000°S -123.500°W", -20.0, -123.5, 12345, "D3N D3E")] // Like this probably unexpected (with the sign), but correct for decimal output
+        [InlineData("20.000°S 123.500°W", -20.0, -123.5, 12345, "U3N U3E")] // Therefore we have this
```

Comment:
why is negative and positive giving same result?

Reply:
or perhaps it's not, can you explain difference between U3N and D3N?

Reply:
"D" is latitude/longitude with sign, "U" is without sign. The sign is typically not used when N/S or E/W is printed. So "D3N" is, while technically possible, not really useful, one would use U3N instead (or just D3 without N)